core/src/main/java/tc/oc/pgm/action/WeatherType.java

```diff
@@ -0,0 +1,7 @@
+package tc.oc.pgm.action;
+
+public enum WeatherType {
```

Comment:
nit:
I'd prefer for this to be inside the `WeatherMatchModule` or in that package